File: Personal Practice.java/one-week-of-trouble/student_grade.py
score = int(input("Enter student score: "))

if score >= 70 and score <= 100:
    print("Grade: A")

elif score >= 60 and score <= 69:
    print("Grade: B")

elif score >= 50 and score <= 59:
    print("Grade: C")

elif score >= 45 and score <= 49:
    print("Grade: D")

elif score >= 0 and score <= 44:
    print("Grade: F")

else:
    print("Invalid score")

File: Personal Practice.java/one-week-of-trouble/StudentGrade.java
import java.util.Scanner;

public class StudentGrade {

    public static void main(String[] args) {

        Scanner input = new Scanner(System.in);

        System.out.print("Enter student score: ");
        int score = input.nextInt();

        if (score >= 70 && score <= 100) {
            System.out.println("Grade: A");
        }

        else if (score >= 60 && score <= 69) {
            System.out.println("Grade: B");
        }

        else if (score >= 50 && score <= 59) {
            System.out.println("Grade: C");
        }

        else if (score >= 45 && score <= 49) {
            System.out.println("Grade: D");
        }

        else if (score >= 0 && score <= 44) {
            System.out.println("Grade: F");
        }

        else {
            System.out.println("Invalid score");
        }
    }
}